Header header
int8 state #2 for ballDetector, 1 for landmarkDetector, 3 for both(for debugging and testing)
